internal/textencoding/truetype.go

```diff
@@ -31,6 +31,7 @@ type TrueTypeFontEncoder struct {
 }
 
 // SubsetRegistered subsets `enc` to only registered runes (that have been registered via encoding).
+// NOTE: Make sure to call this soon before writing (once all needed runes has been registered).
```

Comment:
Minor typo in the note: `has been registered` instead of `have been registered`. 
Same in the note from `model/font.go`.